docs/sources/mimir/configure/configure-shuffle-sharding/index.md

```diff
@@ -129,18 +129,22 @@ If you’re running a Grafana Mimir cluster with shuffle sharding disabled, and
 
 #### Limitation: Decreasing the tenant shard size
 
-The current shuffle sharding implementation in Grafana Mimir has a limitation that prevents you from safely decreasing the tenant shard size when you enable ingesters’ shuffle sharding on the read path.
+The current shuffle sharding implementation in Grafana Mimir has a limitation that prevents you from abruptly decreasing the tenant shard size when you enable ingesters’ shuffle sharding on the read path. Decreasing the tenant shard size without following the complete procedure might result in query failures.
```

Comment:
I know this is not new copy but this part is confusing:
> when you enable ingesters’ shuffle sharding on the read path
This makes it sound like it is part of the process, but it was likely already on for anyone using shuffle sharding and if it was not already on, they would not want to turn it on until after this process is over.

It should be passive to indicate that it has likely already been enabled in the past something "when ingester shuffle sharding is/has been enabled on the read path".